inc/class-command.php

```diff
@@ -61,6 +61,9 @@ class Command {
 	 * [--fields=<fields>]
 	 * : Limit the output to specific fields.
 	 *
+	 * [--<field>=<value>]
```

Comment:
Should we document new parameters in the readme, too?

Reply:
Yes but it will be updated by this Action workflow https://github.com/wp-cli/doctor-command/blob/main/.github/workflows/regenerate-readme.yml